base/src/main/scala/data/DisjunctionFunctions.scala

```diff
@@ -1,10 +1,17 @@
 package scalaz
 package data
 
+import Disjunction.{\/, \/-, -\/}
+
 trait DisjunctionFunctions {
-  @inline def -\/[L](value: L): Disjunction[L, Nothing] = Disjunction.-\/(value)
-  @inline def \/-[R](value: R): Disjunction[Nothing, R] = Disjunction.\/-(value)
+  @inline def -\/[L](value: L): Disjunction[L, Nothing] = -\/(value)
```

Comment:
@S11001001 could (a part of) that convenience be replaced by adding ```toRight```/```toLeft``` methods to ```IdOps```? ie.:
```scala
  final def toLeft[R]: A\/R =
    -\/(self)

  final def toRight[L]: L\/A =
    \/-(self)
```